SPECS-EXTENDED/kernel-lpg-innovate/kernel-lpg-innovate.spec

```diff
@@ -0,0 +1,1959 @@
+# Steps for updating a config file:
+#  1. Extract the linux sources into a folder
+#  2. Add the current config file to the folder
+#  3. Run `make menuconfig` to edit the file (Manually editing is not recommended)
+#  4. Save the config file
+#  5. Copy the config file back into the kernel spec folder
+#  6. Revert any undesired changes (GCC related changes, etc)
+#  8. Build the kernel package
+#  9. Apply the changes listed in the log file (if any) to the config file
+#  10. Verify the rest of the config file looks ok
+# If there are significant changes to the config file, disable the config check and build the
+# kernel rpm. The final config file is included in /boot in the rpm.
+
+# LPG-INNOVATE is x86_64 only, for now
+ExclusiveArch: x86_64
+
+%description
+The kernel package contains the Linux kernel.
+
+%package devel
+Summary:        Kernel Dev
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       gawk
+Requires:       python3
+Obsoletes:      linux-dev
+
+%description devel
+This package contains the Linux kernel dev files
+
+%package drivers-accessibility
+Summary:        Kernel accessibility modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-accessibility
+This package contains the Linux kernel accessibility support
+
+%package drivers-gpu
+Summary:        Kernel gpu modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-gpu
+This package contains the Linux kernel gpu support
+
+%package drivers-intree-amdgpu
+Summary:        Kernel amdgpu modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}-drivers-gpu = %{version}-%{release}
+
+%description drivers-intree-amdgpu
+This package contains the Linux kernel in-tree AMD gpu support
+
+%package drivers-sound
+Summary:        Kernel Sound modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-sound
+This package contains the Linux kernel sound support
+
+%package docs
+Summary:        Kernel docs
+Group:          System Environment/Kernel
+Requires:       python3
+
+%description docs
+This package contains the Linux kernel doc files
+
+%package tools
+Summary:        This package contains the 'perf' performance analysis tools for Linux kernel
+Group:          System/Tools
+Requires:       %{name} = %{version}-%{release}
+Requires:       audit
+
+%description tools
+This package contains the 'perf' performance analysis tools for Linux kernel.
+
+%package grub
+Summary:        Grub configuration for LPG-Innovate kernel
+Group:          System Environment/Kernel
+BuildArch:      noarch
+
+%description grub
+This package contains a grub config file to add required LPG-Innovate parameters to the kernel cmdline.
+
+%package -n     python3-perf
```

Comment:
Can you add a short name to `bpftool` and `python3-perf` packages? Similar to https://github.com/microsoft/azurelinux/blob/3.0-dev/SPECS/kernel-64k/kernel-64k.spec#L6 

This avoids overwriting the kernel's version of these packages

Reply:
ok i added this.

I didn't add any `Conflicts:` (and/or `Provides:`) so the rpm file content of the packages will still conflict, of course.